scripts/nerfcapture2nerf.py

```diff
@@ -0,0 +1,254 @@
+    testbed.nerf.training.set_camera_extrinsics(frame_idx=frame_idx, camera_to_world=X_WV)
+    testbed.nerf.training.set_camera_intrinsics(frame_idx=frame_idx, fx=fx, fy=fy, cx=cx, cy=cy)
+
+
+def live_streaming_loop(reader: DataReader, max_cameras: int):
+    # Start InstantNGP
+    testbed = ngp.Testbed(ngp.TestbedMode.Nerf)
+    testbed.init_window(640, 480)
+    testbed.reload_network_from_file(f"configs/nerf/base.json")
+    testbed.visualize_unit_cube = True
+    testbed.nerf.visualize_cameras = True
+
+    camera_index = 0  # Current camera index we are replacing in InstantNGP 
+    total_frames = 0 # Total frames received
+
+    # Create Empty Dataset
+    testbed.create_empty_nerf_dataset(max_cameras, aabb_scale=1)  
+    testbed.nerf.training.n_images_for_training = 0
+    testbed.up_dir = np.array([1.0, 0.0, 0.0])
+
+    # Start InstantNGP and DDS Loop
+    while testbed.frame():
+        sample = reader.read_next() # Get frame from NeRFCapture
+        if sample:
+            print(f"Frame received")
+
+            # RGB
+            image = np.asarray(sample.image, dtype=np.uint8).reshape(
+                (sample.height, sample.width, 3)).astype(np.float32)/255.0
+            image = np.concatenate(
+                [image, np.zeros((sample.height, sample.width, 1), dtype=np.float32)], axis=-1)
+
+            # Depth if avaiable
+            depth = None
+            if sample.has_depth:
+                depth = np.asarray(sample.depth_image, dtype=np.uint8).view(
+                    dtype=np.float32).reshape((sample.depth_height, sample.depth_width))
+                depth = cv2.resize(depth, dsize=(
+                    sample.width, sample.height), interpolation=cv2.INTER_NEAREST)
+
+
+            # Transform
+            X_WV = np.asarray(sample.transform_matrix,
+                            dtype=np.float32).reshape((4, 4)).T[:3, :]
+
+            # Add frame to InstantNGP
+            set_frame(testbed,
+                    frame_idx=camera_index,
+                    rgb=srgb_to_linear(image),
+                    depth=depth,
+                    depth_scale=1,
+                    X_WV=X_WV,
+                    fx=sample.fl_x,
+                    fy=sample.fl_y,
+                    cx=sample.cx,
+                    cy=sample.cy)
+
+            # Update index
+            total_frames += 1
+            testbed.nerf.training.n_images_for_training = min(total_frames, max_cameras) 
+            camera_index = (camera_index + 1) % max_cameras
+
+            if total_frames == 1:
+                testbed.first_training_view()
+                testbed.render_groundtruth = True
+
+def dataset_capture_loop(reader: DataReader, save_path: Path, overwrite: bool, n_frames: int):
+
+    if save_path.exists():
+        if overwrite:
+            # Prompt user to confirm deletion
+            res = input("Warning, directory exists already. Press Y to delete anyway: ")
+            if res == 'Y':
```

Comment:
This should be written as "[...] Delete anyway? (y/N)" to make clear that the default (pressing enter) corresponds to "no".

Or (my personal preference) is to use (Y/n), i.e. default upon pressing enter is to confirm, like with `colmap2nerf.py`